### ISSUE TITLE
Is still compatible ?

### ISSUE THREAD

**Issue by Jessinra** [wildicedemon/SWAR-X#21]
is it still working on latest SW ? any ban report ?

**Comment by wildicedemon** [comment]
I don't think so, I don't have the time to update this and probably won't in the future the source code is open and I am sure it could be updated. In addition though I doubt anyone would admit it I am sure many of the bots are based on the code contained here. Not that any of them would credit me.
